vendor/pure-css/css/grids-responsive-min.css

```diff
@@ -1,7 +1,7 @@
 /*!
-Pure v0.6.0
-Copyright 2014 Yahoo! Inc. All rights reserved.
+Pure v2.1.0
+Copyright 2013 Yahoo!
```

Comment:
Funny that the copyright is going backward.

Reply:
Huh, I hadn't noticed that. I also updated [the `LICENSE` file](https://github.com/rust-lang/docs.rs/blob/master/vendor/pure-css/LICENSE) but that hadn't changed between versions, and that was already set to 2013.